content/git/concepts/reset/reset.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Reset'
+Description: 'Used to undo commits to a Git repository.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, the `reset` command is used to change the state of the Git repository or undo commits.
+
+## Syntax
```

Comment:
```suggestion
## Syntax

```

Always one empty above and below the headers.

Reply:
Also, let's add 1-2 sentences before the code snippet below, something like "In a terminal, the `reset` command is used in the following way:"